scimitar.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.7.0'
 
-  s.add_dependency 'rails', '~> 7.0'
+  s.add_dependency 'rails', '~> 8.0'
```

Comment:
This means people MUST use Rails 8. We certainly have no intention of upgrading yet given how buggy 7.1 and 7.2 have been for us - it feels like the Rails team is rushing and it's been a very small time between Rails 7 and 8. so we're being very cautious.

I think you want something like [`>= 7.0`](https://bundler.io/guides/gemfile.html) in there?